buffer/src/main/java/io/netty/buffer/AdaptivePoolingAllocator.java

```diff
@@ -886,6 +888,16 @@ protected long unsafeOffset() {
             updater.setInitialValue(this);
             allocator = magazine.parent;
             attachToMagazine(magazine);
+
+            AllocateChunkEvent event = new AllocateChunkEvent();
```

Comment:
Does this mean there is any allocation even if it's not enabled ? Or is the JIT smart enough to eliminate it in this case ?